jelares/portfolio/src/main/java/com/google/sps/servletData/LogInformation.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servletData;
+
+/** Represents a response to the login/logout link on the home page*/
+public final class LogInformation {
+  // Link will say LOGIN if the user is logged out, or LOGOUT if the user is logged in
+  private final String logMessage;
```

Comment:
If this is an enumeration, it seems like using an enum would be clearer and less error-prone, since users of this object would not need to handle string parsing:
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Example:
```
public enum AuthenticationAction {
    LogIn, LogOut 
}

public final class LogInformation {
  public AuthenticationAction getAuthenticationAction();
}
```

Reply:
Changed to an enum similar to the one here